### DISCUSSION TITLE
Game crashes as soon as I click to change her apperience

### DISCUSSION THREAD

**Discussion by ravenlovesmonika** [Monika-After-Story/MonikaModDev#9268] [QA - tech support + general]
So I downloaded the Spritepacks and followed a tutorial. I also gifted everything to her, but whenever I click to change her appearience its just fine for changing outfits, but when it comes to Hairpins/ribbons Monika says "Okay, sure" then the game just instantly crashes.
Does anyone have a solution? (sorry for bad quality pics cant take pics on my pc!)
![Snapchat-601626092](https://user-images.githubusercontent.com/108030849/175141180-c26d0e8e-fc8c-46e5-a103-4b67f30a7c2b.jpg)
![Snapchat-773487848](https://user-images.githubusercontent.com/108030849/175141189-2e9218c7-0d6b-4164-82e6-c3fc9862b332.jpg)

![Snapchat-1662649807](https://user-images.githubusercontent.com/108030849/175141544-3fd8e7cb-7dd6-44a0-a63c-bfc2ff088543.jpg)

**Comment by Booplicate** [comment]
You're missing some mod assets, reinstall the mod. Also consider [moving it out of steam](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#how-do-i-go-from-a-steam-to-a-non-steam-build).

**Reply by ravenlovesmonika** [reply]
reinstalling the mod didnt help, im going to move it out of steam now.
